src/main/resources/com/google/api/codegen/nodejs/package.json.snip

```diff
@@ -41,7 +41,7 @@
     },
     "license": "{@metadata.licenseName}",
     "engines": {
-      "node": ">=4.0.0"
+      "node": ">=6.0.0"
```

Comment:
Are you going 2.0 right now™?

Reply:
We've already made this change across all of the `nodejs-` repositories, figured we may as well here. Happy to back it out if that's a problem :)

Reply:
The generated `package.json` is only used for the new libraries, it won't update the existing ones, so it's safe to put `>=6.0.0` here.

Reply:
Okay, that makes sense to me.